galley/cmd/galley/cmd/root.go

```diff
@@ -86,17 +86,15 @@ func GetRootCmd(args []string) *cobra.Command {
 
 			galleyStop := make(chan struct{})
 
-			if !validationArgs.EnableValidation {
-				go validation.CleanupValidation(galleyStop, validationArgs, kubeConfig, &readinessProbeOptions)
-			}
-
 			if serverArgs.EnableServer {
 				go server.RunServer(serverArgs, livenessProbeController, readinessProbeController)
 			}
 			if validationArgs.EnableValidation {
 				go validation.RunValidation(validationArgs, kubeConfig, livenessProbeController, readinessProbeController)
 			}
 
+			go validation.ReconcileWebhookConfiguration(galleyStop, kubeConfig, validationArgs)
```

Comment:
This is the basic idea. I'll defer to @ozevren for any code organizational issues. 

We may also need an additional flag to disable this reconciliation for some local development and testing modes.

Reply:
Thanks @ayj for your suggestion. @ozevren could you help to review?